source/includes/kubernetes_extension/_k8_configmaps.md

```diff
@@ -7,45 +7,59 @@
+    "data": [
+        {
+            "id": "coredns/kube-system",
+            "data": {
+                "Corefile": ".:53 {\n    errors\n    health\n    kubernetes cluster.local in-addr.arpa ip6.arpa {\n      pods insecure\n      upstream\n      fallthrough in-addr.arpa ip6.arpa\n    }\n    prometheus :9153\n    forward . /etc/resolv.conf\n    cache 30\n    loop\n    reload\n    loadbalance\n    import custom/*.override\n}\nimport custom/*.server\n"
+            },
+            "metadata": {
+                "annotations": {
+                    "kubectl.kubernetes.io/last-applied-configuration": "{\"apiVersion\":\"v1\",\"data\":{\"Corefile\":\".:53 {\\n    errors\\n    health\\n    kubernetes cluster.local in-addr.arpa ip6.arpa {\\n      pods insecure\\n      upstream\\n      fallthrough in-addr.arpa ip6.arpa\\n    }\\n    prometheus :9153\\n    forward . /etc/resolv.conf\\n    cache 30\\n    loop\\n    reload\\n    loadbalance\\n    import custom/*.override\\n}\\nimport custom/*.server\\n\"},\"kind\":\"ConfigMap\",\"metadata\":{\"annotations\":{},\"labels\":{\"addonmanager.kubernetes.io/mode\":\"Reconcile\",\"k8s-app\":\"kube-dns\",\"kubernetes.io/cluster-service\":\"true\"},\"name\":\"coredns\",\"namespace\":\"kube-system\"}}\n"
+                },
+                "creationTimestamp": "2020-06-15T16:14:37.000-04:00",
+                "labels": {
+                    "addonmanager.kubernetes.io/mode": "Reconcile",
+                    "k8s-app": "kube-dns",
+                    "kubernetes.io/cluster-service": "true"
+                },
+                "name": "coredns",
+                "namespace": "kube-system",
+                "resourceVersion": "92",
+                "selfLink": "/api/v1/namespaces/kube-system/configmaps/coredns",
+                "uid": "d77702d6-9f40-40ee-bf2f-b80ff313083f"
+            }
         },
-        "creationTimestamp": "2020-03-20T15:48:41.000-04:00",
-        "name": "cert-manager-cainjector-leader-election",
-        "namespace": "kube-system",
-        "resourceVersion": "25190285",
-        "selfLink": "/api/v1/namespaces/kube-system/configmaps/cert-manager-cainjector-leader-election",
-        "uid": "255fdf58-4f53-4260-abb9-334445c187a2"
-      }
-    }, {
-      etc...
-    } 
-  ]
+        ...
+    ],
+    "metadata": {
+        "recordCount": 4
+    }
+}
 ```
 
-<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/configmaps</code>
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/configmaps?cluster_id=:cluster_id</code>
 
 Retrieve a list of all config maps in a given [environment](#administration-environments).
 
+| Required                   | &nbsp;                                                  |
```

Comment:
Should we add a section like this?
It just looks a bit odd IMO. How do we feel about putting the same content in a 👇 way:
> ***Required<br/>*** `cluster_id`:   The id of the cluster in which to list the config maps.

Reply:
This section matches what's already in pods, released and charts. It also matches the attributes listing right below.